installer/Migration/V5_1_0/Migration.php

```diff
@@ -143,6 +143,25 @@ public function up(): void
         $this->insertSelfReviewWorkflowStates();
         $this->insertReviewListScreenForAdminRole($reviewListScreenId);
         $this->modifyThemeTable();
+
+        $groupId = $this->getLangStringHelper()->getGroupId('general');
+        $toDeleteLangStringId = $this->getLangStringHelper()->getLangStringIdByValueAndGroup('Allows Phone Numbers Only', $groupId);
+        $toPreserveLangStringId = $this->getLangStringHelper()->getLangStringIdByValueAndGroup('Allows numbers and only + - / ( )', $groupId);
+
+
+        $qb = $this->createQueryBuilder()
```

Comment:
isn't it duplicate translation going to exist in DB :thinking:

Reply:
I did a search on the translation yaml files for 5.0 only one each has been added though this was duplicated